spec/Section 3 -- Type System.md

```diff
@@ -371,6 +361,43 @@ that type) then it must not be included.
 When representing a GraphQL schema using the type system definition language,
 all built-in scalars must be omitted for brevity.
 
+**Custom Scalars**
+
+GraphQL services may use custom scalar types in addition to the built-in
+scalars. For example, a GraphQL service could define a scalar called `Time`
+which, while serialized as a string, conforms to ISO-8601. When querying a field
+of type `Time`, you can then rely on the ability to parse the result with an
+ISO-8601 parser and use a client-specific primitive for time. Another example of
+a potentially useful custom scalar is `URL`, which serializes as a string, but
+is guaranteed by the server to be a valid URL.
+
+When defining a custom scalar, GraphQL services should provide a specification
+URL via `specifiedBy` introspection field or `@specifiedBy` directive. This URL 
```

Comment:
You editorialized out a `the` here to make the sentence consistent, but grammatically I think they _both_ need articles? So "via the `specifiedBy` introspection field or the `@specifiedBy` directive."

Reply:
That was a mistake on my part, thanks for the careful read